### ISSUE TITLE
Error thrown when attempting to bind value to input type="hidden" on Safari 13.0.5

### ISSUE THREAD

**Issue by agorf** [alpinejs/alpine#423]
Hello. Thanks a lot for Alpine! It feels great to use, striking the right balance between Stimulus and Vue.

I recently ran into an error when binding `value` (with `:value`) to a hidden input. A TypeError is thrown in Safari (tried it with 13.0.5). No error is thrown in Chrome or Firefox.

Changing the type of the input from `hidden` to `text` seems to stop the error.

Codepen to reproduce: https://codepen.io/agorf/pen/zYvwNGb

The error I get in the console on Codepen:

```
[Error] Unhandled Promise Rejection: TypeError: Type error
	handleAttributeBindingDirective (alpine.js:561)
	(anonymous function) (alpine.js:1512)
	forEach
	resolveBoundAttributes (alpine.js:1498)
	initializeElement (alpine.js:1417)
	(anonymous function) (alpine.js:1401)
	walk (alpine.js:77)
	walk (alpine.js:81)
	walkAndSkipNestedComponents (alpine.js:1379)
	initializeElements (alpine.js:1396)
	Component (alpine.js:1326)
	initializeComponent (alpine.js:1695)
	(anonymous function) (alpine.js:1643)
	(anonymous function) (alpine.js:1659)
	forEach
	discoverComponents (alpine.js:1658)
	(anonymous function) (alpine.js:1642)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJob
```

I'd be glad to provide any other information.

**Comment by keyurshah** [comment]
I"m having a similar error as well when using x-model with an input field which is type `hidden`. 

 Seems to be occurring in Internet Explore 11 and on IOS (iphone) also.

**Comment by Roywcm** [comment]
I have the same issue

**Comment by agorf** [comment]
Fixed with #453
